### DISCUSSION TITLE
How does wBTC stay pegged to BTC ?

### DISCUSSION THREAD

**Discussion by TilakMaddy** [Cyfrin/foundry-full-course-cu#2719] [Q&A]
Does anyone know how wBTC stays pegged to BTC ? 
I know that USDC is regulated by govt officials. (maybe I'm wrong here too) but no-one seems to talk about wBtc.  

I would imagine the bridge is a lot weaker because they are ethereum nodes are a lot different from bitcoin nodes so you don't really get to leverage zksync style bridging.

I Appreciate any insight : )

**Comment by EngrPips** [comment]
I have never done any research on this, but I will be glad to hear whatever you find out @TilakMaddy.

**Comment by TilakMaddy** [comment]
Turns out I googled everything, but "wrapped bitcoin" haha

Here it explains -> there is a DAO that votes for the value of wbtc  https://www.wbtc.network/

**Reply by EngrPips** [reply]
Interesting, Thanks for sharing this with me buddy.
